config_gen.py

```diff
@@ -216,6 +218,20 @@ def run(cmd, *args, **kwargs):
         else:
             run([make_cmd, "maintainer-clean"], env=env, **proc_opts)
 
+    elif(glob.glob(os.path.join(project_dir, "*.pro"))):
+        # qmake
```

Comment:
I thought I'd use the upstream capitalization, but now I saw Autotools is capitalized as well - will do.